sqlx-core/src/postgres/types/ltree.rs

```diff
@@ -27,13 +27,17 @@ pub enum PgLTreeParseError {
 pub struct PgLTreeLabel(String);
 
 impl PgLTreeLabel {
-    pub fn new(label: &str) -> Result<Self, PgLTreeParseError> {
+    pub fn new<S>(label: S) -> Result<Self, PgLTreeParseError>
+    where
+        S: Into<String>,
```

Comment:
I believe this change too can be seen as a breaking change

Reply:
Yep, this can break type inference, so it's breaking.

It shouldn't matter since the `push` change is breaking anyways and pushes this patch to the next breaking release